nmn.Templates/README.md

```diff
@@ -109,5 +110,28 @@ The `*|TITLE|*`, `*|START|*` and `*|END|*` can be mixed with whatever markdown c
 
 You can also place  `{{matchingEvents()}}` or `{{listMatchingEvents()}}` in Templates in a similar way, and similar customisation is possible. However, it is defined in a different way, using the matches and template strings defined in the \_configuration file's `addMatchingEvents` array, as shown above.
 
+### weekDates - ouput the beginning and end span/dates of the current week (e.g. Mon-Sun)
+-  `{{weekDates({weekStartsOn:1, format:"EEE yyyy-MM-dd"})}}` -- get the span of this week 
+	- e.g. `Mon 2021-08-30 - Sun 2021-09-05` (mon-sun by default) 
+	- weekStartsOn parameter is 1 (Monday) by default. Change it to 0 (Sunday) or another day to start the weeks on a different day
+	- see [date-fns format](https://date-fns.org/v2.23.0/docs/format) for formatting details.
+
+## `/qtn - Quick Template Note` 
```

Comment:
I found this in the `nmn.Templates/plugin.json` but there is no short command, thus it won't execute inline or via command bar).  I would suggest adding `qtn` as an alias (or maybe I am using wrong as I have never used the `nmn.Templates` plugin

```json
    {
      "name": "qtn - Quick template note",
      "alias": ["quick", "template"],
      "description": "Choose a note and create it based on _configuration settings quickNotes",
      "jsFunction": "quickTemplateNote"
    } 
```